### DISCUSSION TITLE
Do I need to change hasExternalScripts to true for Analytics to work?

### DISCUSSION THREAD

**Discussion by dmastag** [arthelokyo/astrowind#455] [Q&A]
I noticed that partytown is not properly configured unless I change hasExternalScript to True.
Is that intended?

https://github.com/onwidget/astrowind/blob/main/astro.config.mjs#L23

**Comment by shebinleo** [comment]
@dmastag Yes, the Google Analytics only worked after setting `hasExternalScripts` to `true`.
